manifests/init.pp

```diff
@@ -66,6 +67,12 @@
     locale                     => 'en_US.utf8',
   }
 
+  # Starting with PE 2016.1.1; a minimum of 200 max_connections
+  # is recommended
+  postgresql::server::config_entry { 'max_connections':
```

Comment:
This should use postgresql_conf as seen here:

https://github.com/npwalker/pe_databases/blob/master/manifests/postgresql_settings.pp#L75-L77